runtime/src/transaction_priority_details.rs

```diff
@@ -27,11 +27,11 @@ pub trait GetTransactionPriorityDetails {
         let prioritization_fee_details = compute_budget
             .process_instructions(
                 instructions,
-                true,  // use default units per instruction
-                false, // stop supporting prioritization by request_units_deprecated instruction
-                true,  // enable request heap frame instruction
-                true,  // enable support set accounts data size instruction
-                       // TODO: round_compute_unit_price_enabled: bool
+                true, // use default units per instruction
+                true, // supports prioritization by request_units_deprecated instruction
```

Comment:
Kinda worried about putting true here.
Before we remove the code but after the feature is activated this means all txs using this unsupported ix get boosted to the front of our queue w/o hope of ever getting executed or paying fees (they will later be dropped when we try to execute).

Reply:
To clarify, after feature gate is enabled, and before code is removed. A TX with deprecated ix would: 
1. be placed into priority queue based on priority specified by deprecated ix
2. then being dropped by leader threads during packing (which is before loading and execution).

Later after feature code is removed, the same transaction will not be inserted into priority queue. 

So there will be some queuing inefficiency during the window of activated feature code removal.

Reply:
> then being dropped by leader threads during packing (which is before loading and execution).

I think it'd be dropped during execution - after locks are taken and as first step in execution.